pkg/cloud/alibaba/provider.go

```diff
@@ -49,6 +49,7 @@ const (
 	ALIBABA_UNKNOWN_INSTANCE_FAMILY_TYPE       = "unknown"
 	ALIBABA_NOT_SUPPORTED_INSTANCE_FAMILY_TYPE = "unsupported"
 	ALIBABA_DISK_CLOUD_ESSD_CATEGORY           = "cloud_essd"
+	ALIBABA_DISK_CLOUD_EFFICIENCY_CATEGORY     = "cloud_efficiency"
```

Comment:
are you using this const anywhere? did you need it for your new test?

Reply:
I'm sorry, I forgot to delete it. I did use this variable in the initial test.